integration-tests/src/main/java/org/apache/druid/testing/utils/ITRetryUtil.java

```diff
@@ -30,7 +30,7 @@
 
   private static final Logger LOG = new Logger(ITRetryUtil.class);
 
-  public static final int DEFAULT_RETRY_COUNT = 120; // 10 minutes
+  public static final int DEFAULT_RETRY_COUNT = 240; // 20 minutes
```

Comment:
Can you explain how we decided to set this to 20 minutes instead of 10?

Reply:
We don’t really have any fancy calculation to determine this. It’s more of a try and see and adjust loop. For example, ‘DEFAULT_RETRY_SLEEP’ used to be 10 secs, then it was changed to 2secs but kafka IT didn’t pass consistently so it was increased to 5secs